src/groups/group_action.jl

```diff
@@ -0,0 +1,90 @@
+"""
+    AbstractActionOnManifold
+
+An abstract group action on a manifold.
+"""
+abstract type AbstractActionOnManifold end
+
+"""
+    base_group(A::AbstractActionOnManifold)
+
+The group that acts in action `A`.
+"""
+function base_group(A::AbstractActionOnManifold)
+    error("Function base_group is not yet defined for type $(typeof(A)).")
```

Comment:
Can we keep these errors consistent with those in ManifoldsBase? e.g. this would be 
```julia
error("base_group not implemented for type $(typeof(A)).")
```